### PR TITLE
Add message when construction finished.

### LINKED ISSUE #22292: Add record to log when construction finishes
A minor thing: would be nice to record in the activity log when construction finishes.

We add a record log for most other player actions ("You drop your matchbook on the grass."). Having a recorded instance of you finishing construction would be a nice QoL addition. At the very least, helps with understanding what's happened since that single instance.

A good beginning might simply be "You finish your construction," since the verbs relating to each of the individual constructions are varied ("you finish digging the pit," "you finish boarding up the window", etc).

Idea from @Asmageddon in #19930

### PR DESCRIPTION
**Pull request by laeaton2001**
Fixes #22292
Verb from json is now shown when the constructoin is complete.

Screenshot:
image

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -2251,7 +2402,8 @@
         ],
         "pre_note" : "Can be deconstructed without tools.",
         "pre_special" : "check_empty",
-        "post_terrain" : "f_kiln_empty"
+        "post_terrain" : "f_kiln_empty",
+        "verb": "buidling the charcoal kiln"
```
Comment (Night-Pryanik) [inline data/json/construction.json]: "buildling" -> "building".
The same below.

**Comment by Coolthulhu** [comment]
Would be better if most constructions had messages like:
"You complete the construction: Doing The Thing"

Most of the messages here aren't any more informative than the above, but they still require translating and keeping around.
For those selected few (if any) which require a different message, the current message isn't good enough either, since it is hard to translate due to the string splice. In those cases, you can't just supply a verb, you have to supply the entire message. But again, this should only cover the exceptions.

**Comment by BevapDin** [comment]
What Coolthulhu said, and a hint that applies to all future JSON expansion: please add new object entry somewhere *before* that last entry.

Because now the diff shows two lines have changed for each object: one line (the previously last entry) has changed by adding a singe comma, and the second line has changed as it's new.

Adding the new entry somewhere else means only one line changes: the one line that has been added:

```diff
 "some_object" : {
   "one" : "value",
-  "two" : "old value"
+  "two" : "old value",
+  "three" : "new entry"
 }
```
Better (and equal because JSON object members are unsorted):
```diff
 "some_object" : {
   "one" : "value",
+  "three" : "new entry",
   "two" : "old value"
 }
```

**Comment by laeaton2001** [comment]
Thanks for the feedback, I'll change this once I'm back at my dev pc.

**Comment by laeaton2001** [comment]
I think I did that right, not used to reverting in git.

**Pull request merged**